RELEASE/scripts/autoscend/auto_mr2020.ash

```diff
@@ -117,6 +117,20 @@ boolean auto_powerfulGloveNoncombat()
 	return ret;
 }
 
+int auto_powerfulGloveReplacesPossible()
+{
+	if (!auto_hasPowerfulGlove()) return 0;
+
+	return (auto_powerfulGloveCharges() / 10).to_int();
+}
+
+int auto_powerfulGloveReplacesAvailable()
```

Comment:
These are good utility functions, perhaps change these function names/add a comment to make clear that one is speculative out-of-combat and one is "use this in combat"?

Reply:
yeah I'll rename it